client/components/select-vertical/style.scss

```diff
@@ -0,0 +1,73 @@
+/**
+ * Select vertical
+ */
+
+.select-vertical__suggestion-search {
+    position: relative;
+    width: 504px;
+
+    &.is-focused {
+        .select-vertical__suggestion-search-dropdown {
+            border-color: #646970;
+            box-shadow: 1px 1px 0 1px #e2eaf1, -1px 1px 0 1px #e2eaf1;
+        }
+    }
+
+    &.is-show-suggestions {
+        &.is-focused {
+            input.form-text-input[type='text'] {
+                box-shadow: 0 0 0 2px #e2eaf1;
+            }
+        }
+    }
+
+    input.form-text-input[type='text'] {
+        border-radius: 4px;
+        height: 44px;
+        font-size: $font-body;
+        line-height: 20px;
+
+        &::placeholder {
+            font-size: $font-body-small;
+        }
+
+        &:focus {
+            border-color: #646970;
+            box-shadow: none;
+        }
+    }
+
+    @media screen and ( max-width: 600px ) {
+        width: auto;
+    }
+}
+
+.select-vertical__suggestion-search-dropdown {
+    border-bottom-left-radius: 4px;
+    border-bottom-right-radius: 4px;
+    border-color: var( --color-neutral-10 );
+    border-style: solid;
+    border-width: 0 1px 1px;
+    box-sizing: border-box;
+    margin-top: -2px;
+    transition: ease-in-out 0.15s all;
```

Comment:
could we specify the transition property if possible?

Reply:
Updated